pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -39,15 +64,39 @@ type provider interface {
 type Service struct {
 	outboundTransport transport.OutboundTransport
 	store             storage.Store
+	callbackChannel   chan didCommChMessage
```

Comment:
Suggest use of "inbound/outbound" or "incoming/outgoing" prefixes to add clarity:

* callbackChannel -> inboundCallbacks
* actionEvent -> outboundActions
* streamEvents -> outboundEvents

`execute` could use a rename to `autoExecute`

Reply:
execute doesn't mean autoExecute (which also means it could use a rename to something more explicit.)